plugins/outputs/redistimeseries/redistimeseries.go

```diff
@@ -0,0 +1,90 @@
+package redistimeseries
+
+import (
+	"fmt"
+
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  address = "127.0.0.1:6379"
+
+  ## Redis ACL credentials
+  # username = ""
+  # password = ""
+  # database = 0
```

Comment:
I still don't see `username` and `database` reflected in the README file, `sampleConfig` and the README example should match as this is what the user sees when viewing your plugin in github. Am I missing something?

Reply:
@srebhan I just added some readme updates as per your ask. If there's something else you'd like to see, or that I inadvertently missed, please let me know!

Reply:
I added a diff, so maybe it becomes more clear on what I'm after. :-)

Reply:
Thank you. I'm new to the project - this is helpful!